modules/services/nix-gc/default.nix

```diff
@@ -35,9 +36,12 @@ in
       };
 
       interval = mkOption {
-        type = types.attrs;
+        type = let
+          t = launchdConfig.options.StartCalendarInterval.type;
+        in
+          types.addCheck (t // {description = lib.removePrefix "null or" t.description;}) (x: x != null);
```

Comment:
Thank you, this is great! Would you mind making a `modules/launchd/types.nix` and exporting the type without the `nullOr` from it, so that it can be used directly in both `launchd.nix` and `nix-gc/default.nix` instead of having to do this hack?